app/HutchAgent/HutchAgent.csproj

```diff
@@ -12,6 +12,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="YamlDotNet" Version="13.1.0" />
```

Comment:
not sure sure which commit this got added, but can we remove since it's part of another PBI pls?